build.assets/Makefile

```diff
@@ -381,6 +439,58 @@ enter-root: buildbox
 		-e HOME=$(SRCDIR)/build.assets -w $(SRCDIR) $(BUILDBOX) /bin/bash
 
 #
+# Reconfigure crosstool-NG for the given ARCH via its menuconfig system.
+# e.g.
+# $ make config/buildbox-ng CTNG_ARCH=arm64
+#
+# After saving and exiting in the menuconfig, you should have an updated
+# defconfig file in build.assets/buildbox/ct-ng-configs
+#
+# You need to have a local copy of the configuration container image
+# created with: make buildbox-thirdparty-ctng
+#
+.PHONY: config/buildbox-ng
+config/buildbox-ng:
+	@: $(or $(CTNG_ARCH),$(error CTNG_ARCH variable must be set))
+	@: $(or $(wildcard buildbox/ct-ng-configs/$(CTNG_ARCH).defconfig),\
+		$(error No config for arch $(CTNG_ARCH)))
+	docker run -it --rm \
+		--hostname $(HOSTNAME) \
```

Comment:
Why do we need hostname here?

Reply:
This is what was done with the other buildboxes: https://github.com/gravitational/teleport/blob/520539631dd973c2b3d4265250ad81803242bda4/build.assets/Makefile#L11

I can't use the `$(DOCKERFLAGS)` var as it has stuff I don't want so I just took the other bits. I'm ambivalent to setting hostname and am happy to remove this.

Reply:
If the docker need it I'd leave it and comment on why it's needed. If we can build without setting it I'd remove it.

Reply:
We don't need it. I think someone perhaps just liked it. I've removed it.